packages/amplify/amplify_flutter_android/android/build.gradle

```diff
@@ -81,6 +81,6 @@ dependencies {
     testImplementation 'androidx.test:core:1.4.0'
     testImplementation 'org.robolectric:robolectric:4.3.1'
     testImplementation 'com.google.code.gson:gson:2.8.6'
-    testImplementation 'com.amplifyframework:aws-auth-cognito:1.36.1'
+    testImplementation 'com.amplifyframework:aws-auth-cognito:v1.37.0-cpkey-preview.2'
```

Comment:
Will this branch be up to date with recent fixes from amplify-android?

Reply:
Yes it's based on amplify-android main.